Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -869,9 +870,8 @@ void GetBPRegInfo(const u8* data, std::string* name, std::string* desc)
   case BPMEM_TREF + 5:
   case BPMEM_TREF + 6:
   case BPMEM_TREF + 7:
-    SetRegName(BPMEM_TREF);
-    // TODO: Description
-    break;
+    return std::make_pair(fmt::format("BPMEM_IREF number {}", cmd - BPMEM_TREF),
```

Comment:
Should say `BPMEM_TREF`.  The `RegName` macro usually covers things like this (though I mainly used a macro like that because one was used before; I'm not fully convinced it really helps), but I didn't use it in cases where `fmt` was used.